package net.sf.freecol.tools;

import org.junit.*;
import static org.junit.Assert.*;
import org.xml.sax.SAXParseException;

/**
 * The class <code>SaveGameValidatorTest</code> contains tests for the class <code>{@link SaveGameValidator}</code>.
 *
 * @generatedBy CodePro at 5/10/16 11:07 AM
 * @author newsha
 * @version $Revision: 1.0 $
 */
public class SaveGameValidatorTest {
	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_1()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_2()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_3()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_4()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_5()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_6()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_7()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_8()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_9()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_10()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_11()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_12()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_13()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_14()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_15()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Test(expected = org.xml.sax.SAXParseException.class)
	public void testMain_16()
		throws Exception {
		String[] args = new String[] {};

		SaveGameValidator.main(args);

		// add additional test code here
	}

	/**
	 * Perform pre-test initialization.
	 *
	 * @throws Exception
	 *         if the initialization fails for some reason
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@Before
	public void setUp()
		throws Exception {
		// add additional set up code here
	}

	/**
	 * Perform post-test clean-up.
	 *
	 * @throws Exception
	 *         if the clean-up fails for some reason
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	@After
	public void tearDown()
		throws Exception {
		// Add additional tear down code here
	}

	/**
	 * Launch the test.
	 *
	 * @param args the command line arguments
	 *
	 * @generatedBy CodePro at 5/10/16 11:07 AM
	 */
	public static void main(String[] args) {
		new org.junit.runner.JUnitCore().run(SaveGameValidatorTest.class);
	}
}